File: sandbox/dotnet/src/Dropbox.SignSandbox/FaxLineRemoveUserDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class FaxLineRemoveUserDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();

        var faxLineRemoveUserRequest = new FaxLineRemoveUserRequest(
            number: "[FAX_NUMBER]",
            emailAddress: "member@dropboxsign.com"
        );

        try
        {
            var response = new FaxLineApi(config).FaxLineRemoveUser(
                faxLineRemoveUserRequest: faxLineRemoveUserRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling FaxLine#FaxLineRemoveUser: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: sandbox/ruby/src/FaxLineRemoveUserDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

fax_line_remove_user_request = Dropbox::Sign::FaxLineRemoveUserRequest.new
fax_line_remove_user_request.number = "[FAX_NUMBER]"
fax_line_remove_user_request.email_address = "member@dropboxsign.com"

begin
    response = Dropbox::Sign::FaxLineApi.new.fax_line_remove_user(
        fax_line_remove_user_request,
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling FaxLine#fax_line_remove_user: #{e}"
end
